File: Zopoesht.Application/AdministrativeExpenses/Dtos/AnnualAdministrativeExpensesHistoryDto.cs
using Zopoesht.Application.Nomenclatures.Dtos;

namespace Zopoesht.Application.AdministrativeExpenses.Dtos
{
    public class AnnualAdministrativeExpensesHistoryDto
    {
        public int UserId { get; set; }

        public AuthorityDto UserAuthority { get; set; }

        public string UserFullName { get; set; }

        public DateTime ModificationDate { get; set; }

        public decimal AnnualAdministrativeExpenseAmount { get; set; }
    }
}

File: WebApplication/src/modules/annual-administartive-expenses/dtos/annual-administrative-expenses-history.dto.ts
import { AuthorityDto } from "../../nomenclature/dtos/authority.dto";

export class AnnualAdministrativeExpensesHistoryDto {
  userAuthority: AuthorityDto;
  userFullName: string;
  modificationDate: Date;
  annualAdministrativeExpenseAmount: number;
}